.github/workflows/moodle-plugin-ci.yml

```diff
@@ -29,9 +29,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.3', '7.4', '8.0']
+        php: ['7.4']
```

Comment:
Dear @abias why would you drop PHP 8.0 testing?

Reply:
I guess for fater testlng, then you would add it back in again, right?

Reply:
Exactly. I am currently working on the last tests and then this is ready for a final review.